src/pages/guidelines/2x-grid/implementation.mdx

```diff
@@ -0,0 +1,1264 @@
+
+![Do align components within containers flush to the columns.](images/16px_do.png)
+
+  </DoDontExample>
+</Column>
+<Column colMd={4} colLg={4} >
+  <DoDontExample caption="Do not hang type into the gutter under any circumstances.">
+
+![Do not hang type into the gutter under any circumstances.](images/16px_dont.png)
+
+  </DoDontExample>
+</Column>
+</Row>
+
+### The Condensed grid
+
+The Condensed grid collapses the gutter to 2px. To achieve this, containers and components bleed 16px into the gutter on either side of the column. This grid mode is most often used in conjunction with another grid mode, either Cozy or Comfortable, and arguably requires the most design skill to incorporate into layouts.
+
+Rows that are adjacent will have a 2px of margin between them to match gutter. Use vertical helpers here.
+
+<Row>
+  <Column noGutterSm>
+    <div className="grid-demo grid-demo--condensed">
+      <div class="bx-demo--grid bx-demo--grid--condensed">
+        <div class="bx-demo--row">
+          <div class="bx-demo--col"><div class="grid-demo-content"></div></div>
+          <div class="bx-demo--col"><div class="grid-demo-content"></div></div>
+          <div class="bx-demo--col"><div class="grid-demo-content"></div></div>
+          <div class="bx-demo--col "><div class="grid-demo-content"></div></div>
+        </div>
+      </div>
+    </div>
+
+```html
+<div class="bx--grid bx--grid--condensed">
+  <div class="bx--row">
+    <div class="bx--col"></div>
+    <div class="bx--col"></div>
+    <div class="bx--col"></div>
+    <div class="bx--col"></div>
+  </div>
+</div>
+```
+
+  </Column>
+</Row>
+
+<br />
+
+#### Usage
+
+When you have separate pieces of information that form a larger picture, like a dashboard or an overview page, that’s a good time to use the Condensed grid.
+You'll also see this grid mode in portals, resource tiles and other introductory UI. See it in action on the [IBM Design Language homepage](https://www.ibm.com/design/language/).
+
+As mentioned above, fixed components with labels, such a input fields and dropdowns, should not hang into the gutter.
+
+![2px gutter](images/2px_gutter.png)
+
+<Row>
+<Column colMd={4} colLg={4}>
+  <DoDontExample type="do" caption="Do keep all type (inside and outside containers) aligned with the column grid.">
+
+![Do keep all type (inside and outside containers) aligned with the column grid.](images/2px_do.png)
+
+  </DoDontExample>
+</Column>
+<Column colMd={4} colLg={4} >
+  <DoDontExample caption="Do not hang type into the gutter under any circumstances.">
+
+![Do not hang type into the gutter under any circumstances.](images/2px_dont.png)
+
+  </DoDontExample>
+</Column>
+</Row>
+
+### Mix and match
+
+You can use rows to mix and match grid modes. In this situation, margin bottom between the rows can vary. Because of input fields with labels, both the Cozy grid mode and the Condensed grid are most often used in conjunction with the Comfortable grid mode.
```

Comment:
"Because of input fields with labels..." maybe explain what about inputs. Like "Because of scenarios like input fields where the label text doesn't align to the input text, both the ..."

Reply:
All of these changes have been made!